code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -297,6 +322,11 @@
 	icon_state = "maintenance"
 	base_icon_state = "maintenance"
 
+/obj/item/reagent_containers/hypospray/medipen/pumpup/examine()
+	. = ..()
+	if(length(reagents?.reagent_list))
+		. += span_notice("Theres a small LCD screen on the side of the medipen which reads, 'ERROR: Unidentified chemical contained inside medipen, use with caution!' in small text.")
```

Comment:
Same here

Reply:
Yeah I think I could but there just doesn't seem to be a reason to, it'd bulk up the code some more all to make it so I don't have to write two lines of code which I already have wrote, both of which have different examine texts then the other, so I'd have to write two different true/false statements, one for the glitching LCD screen and one for the old medipen examine... At least I think, I could be totally wrong

Reply:
Make it a var on the parent to avoid showing parent's description

Reply:
Ok, yeah sure Ill do that in a few hours